ONBOARDING.md

```diff
@@ -17,6 +17,7 @@ invalid and are superseded by the TSC Charter.
 * Add them to the `@pkgjs` org and with an `Owner` role.
 * Add them to the `@pkgjs/node-tsc` team.
 * Invite them to the HackerOne [Node.js team](https://hackerone.com/nodejs/team_members).
+  Have them turn on two-factor authentication after they have joined.
```

Comment:
Is this supposed to be a new bullet or a continuation of the HackerOne instructions?

Reply:
A continuation.